phpunit.xml.dist

```diff
@@ -2,18 +2,22 @@
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          colors="true"
          bootstrap="vendor/autoload.php"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.5/phpunit.xsd">
-  <coverage>
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.1/phpunit.xsd">
+  <coverage/>
+
+  <testsuites>
```

Comment:
Can this tag be kept in the previous position? Just to minimize changes

Reply:
I have reverted the config-file. (see below)